target/linux/meson/patches-5.15/903-Support-for-EMMC.patch

```diff
@@ -0,0 +1,58 @@
+--- a/arch/arm/boot/dts/Makefile
++++ b/arch/arm/boot/dts/Makefile
+@@ -365,6 +365,7 @@ dtb-$(CONFIG_MACH_MESON8) += \
+ 	meson8b-ec100.dtb \
+ 	meson8b-mxq.dtb \
+ 	meson8b-odroidc1.dtb \
++	meson8b-onecloud.dtb \
```

Comment:
i dont think this is related to "support for emmc"

split this change to `900-arm-boot-add-dts-files.patch`